### PR TITLE
Update control panel links and info for Cloud Load Balancers: INFODEV-2693

### PR DESCRIPTION
**Pull request by RackerKate**
@cyrichardson Please review - ready to go.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-56ed4e99fc/how-to/)

**Comment by cyrichardson** [comment]
@RackerKate on this one now.

File: content/cloud-load-balancers/pointing-domains-to-load-balancers.md [hunk 1]
```diff
@@ -1,46 +1,50 @@
 ---
 permalink: pointing-domains-to-load-balancers/
 audit_date:
-title: Point Domains to Load Balancers
+title: Point Domains to load balancers
```
Comment (cyrichardson) [inline content/cloud-load-balancers/pointing-domains-to-load-balancers.md]: Point domains to load balancers

File: content/cloud-load-balancers/pointing-domains-to-load-balancers.md [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
-last_modified_date: '2016-04-21'
-last_modified_by: Rose Contreras
+last_modified_date: '2018-10-24'
+last_modified_by: Kate Dougherty
 product: Cloud Load Balancers
 product_url: cloud-load-balancers
 ---
 
 You can load balance incoming traffic to your domain using a Cloud Load
 Balancer. Suppose you initially have one Cloud Server with a domain
```
Comment (cyrichardson) [inline content/cloud-load-balancers/pointing-domains-to-load-balancers.md]: Balancers.

File: content/cloud-load-balancers/pointing-domains-to-load-balancers.md [hunk 3]
```diff
@@ -1,46 +1,50 @@
...
-pointing to the public IP Address of the server. Then traffic to your
-domain starts picking up and suddenly overloads your server. To improve
-traffic flow, you can create a clone of the server and then put both
+pointing to the public Internet Protocol (IP) Address of the server. Then
```
Comment (cyrichardson) [inline content/cloud-load-balancers/pointing-domains-to-load-balancers.md]: Internet Protocol (IP) address

File: content/cloud-load-balancers/pointing-domains-to-load-balancers.md [hunk 4]
```diff
@@ -1,46 +1,50 @@
...
+traffic to your domain starts picking up and suddenly overloads your server.
+To improve traffic flow, you can create a clone of the server and then put both
 servers behind a new load balancer. Instead of pointing your domain
 directly to the old server, you can now point it to the new load
-balancer, which will distribute the traffic evenly and prevent server
+balancer, which distributes the traffic evenly and prevents server
 overload. Additionally, you could create several new domains that
```
Comment (cyrichardson) [inline content/cloud-load-balancers/pointing-domains-to-load-balancers.md]: Additionally, you can create

File: content/cloud-load-balancers/products-i-can-load-balance.md [hunk 5]
```diff
@@ -5,24 +5,25 @@ title: Products I Can Load Balance
...
+server, database server, website, or your personal web server at
 home. In load balancing terminology, the resources that you can load
 balance are called *nodes*.
 
 ### Adding nodes
 
-When you create a load balancer using the Control Panel you'll see that
-Add Nodes is part of the configuration process:
+When you create a load balancer by using the [Cloud Control
+Panel](https://login.rackspace.com) you see that Add Nodes is part of the
```
Comment (cyrichardson) [inline content/cloud-load-balancers/products-i-can-load-balance.md]: anel](https://login.rackspace.com), you

File: content/cloud-load-balancers/products-i-can-load-balance.md [hunk 6]
```diff
@@ -34,4 +35,5 @@ region of the load balancer can impact performance of the load balancer.
 We recommend creating the load balancer in the region that is closest to
 your external nodes.
 
-For more information, see [Load Balancing Public Vs. Private IPs](/how-to/load-balancing-internal-ips-in-the-same-region).
+For more information, see [Load Balancing Public Vs. Private
```
Comment (cyrichardson) [inline content/cloud-load-balancers/products-i-can-load-balance.md]: [Load Balancing Internal IPs in the same region](/how-to/load-balancing-internal-ips-in-the-same-region).

File: content/cloud-load-balancers/using-the-rackspace-network-effectively.md [hunk 7]
```diff
@@ -5,25 +5,25 @@ title: Using the Rackspace network effectively
...
-Load Balancers, and Cloud Databases) located in the same region, you can
-use the internal IP address to connect the infrastructure together. For
-example, you can create a Cloud Load Balancer for two or more Cloud
-Servers in the same region using the Rackspace Network internal IP
+Rackspace Network Internet Protocol (IP) address. This IP address is not
+visible on the public internet. However, if you have other assets (Cloud
+Servers, Cloud Load Balancers, and Cloud Databases) located in the same
+region, you can use the internal IP address to connect the infrastructure
+together. For example, you can create a Cloud Load Balancer for two or more
+Cloud Servers in the same region using the Rackspace Network internal IP
```
Comment (cyrichardson) [inline content/cloud-load-balancers/using-the-rackspace-network-effectively.md]: region by using

File: content/cloud-load-balancers/using-the-rackspace-network-effectively.md [hunk 8]
```diff
@@ -36,18 +36,19 @@ offer the first level of network security. For example if you're not
...
 firewall around them and reduce their exposure to outside threats. Note
 that Rackspace creates all new servers with basic security settings,
-however you should follow the instructions located in [Configuring Basic Security](/how-to/configuring-basic-security) to
+however you should follow the instructions located in [Configuring Basic
+Security](/how-to/configuring-basic-security) to
 further protect your data.
 
 ### Finding the Rackspace IP address
 
 You can find all of the IP addresses for a new server, load balancer or
```
Comment (cyrichardson) [inline content/cloud-load-balancers/using-the-rackspace-network-effectively.md]: load balancer, or

File: content/cloud-load-balancers/using-the-rackspace-network-effectively.md [hunk 9]
```diff
@@ -36,18 +36,19 @@ offer the first level of network security. For example if you're not
...
-database by clicking on it and viewing its Details.
+database in the [Cloud Control Panel](https://login.rackspace.com) by clicking on it and viewing its Details.
```
Comment (cyrichardson) [inline content/cloud-load-balancers/using-the-rackspace-network-effectively.md]: its details.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b82bb45d80/how-to/). :bow:

**Pull request merged**